Python 3 Update

DomAmato opened the pull request:
Not sure if you are actively maintaining this code base anymore.

Here is a python 3 update

If not would you be ok with either transfering or allowing a hard fork (like pycrypto and pycryptodome did)

DomAmato committed:
Convert to Python 3 (#1)

theduderog commented:
Hi @DomAmato, I'm not able to maintain it anymore  Happy to transfer it.

DomAmato commented:
cool, let me know if there is anything you need from me but you can just transfer it to my personal account and I can move it to a final destination.

laf0rge commented:
Hi @theduderog, @DomAmato, is there any progress about the transfer?  There is a working python3 port around for years, but it is not installable via pip on python3, as the last version in pypi is version 0.3 (python 2.7 only) from 9 years ago.

Thanks for any help in unblocking this.

DomAmato committed:
Fix error when reusing the encoder for decode/encode  (#2)

* Fix bug in length value not resetting after decoding

also added a type for network error codes to better show what they are, even though the spec is not honored in most implementations.

* remove old python versions

* add a test to do decode and encoding with the same encoder

* add test to check if patch works

uncommenting the code fix causes the test to fail

* make field names of named tuples easier to read
